browser/src/control/Control.JSDialogBuilder.js

```diff
@@ -1455,6 +1455,18 @@ L.Control.JSDialogBuilder = L.Control.extend({
 		if (data.placeholder)
 			$(edit).attr('placeholder', data.placeholder);
 
+		edit.addEventListener('keydown', function() {
+			if (edit.value === data.text) {
+				edit.value = '';
+			}
+		});
+
+		edit.addEventListener('focusout', function() {
+			if (edit.value === '') {
+				edit.value = data.text;
+			}
+		});
```

Comment:
This will be applied to any edit field. I can imagine there are places we don't want that behaviour.
Also it can break getting the real information because it will provide us old value if content was deleted when we close dialog. I think this should be done in Zotero dialog implementation in the callback... only for this specific case.

Reply:
understood, so I will remove this but it would be great to have this (the possibility to have a placeholder that goes away when you start type, actually how it happens on Zotero app)

Reply:
removed that last commit